src/Sylius/Bundle/CoreBundle/Resources/config/app/main.yml

```diff
@@ -157,6 +157,6 @@ jms_serializer:
                 path: "@SyliusVariationBundle/Resources/config/serializer"
 
 a2lix_translation_form:
-    locale_provider: sylius.locale_provider
+    locale_provider: sylius.a2lix_locale_provider
```

Comment:
A problem that we were facing was that in administration panel we expect to see all enabled (not current channel's) locales, because admin users want to add translations to all languages. But because a2lix locale provider was channel-aware it would only show current channel's locales.

Can you verify that this change is not a regression on this matter? 
https://github.com/Sylius/Sylius/pull/2945